README.md

```diff
@@ -82,3 +82,54 @@ When updating the config.template and txt files in tests/fixtures make sure to r
 This repository deprecates the following repositories:
 * https://github.com/NebraLtd/snapshot-bumper
 * https://github.com/NebraLtd/vps-scripts
+
+## Debugging
+
+1. After connecting via SSH, you will need to change to change to root for most opertions:
+
+```
+marvin@snapshotter:~$ sudo su
+root@snapshotter:/home/marvin#
+```
+
+2. Check miner container status:
+
+```
+# docker ps
+CONTAINER ID   IMAGE                                                   COMMAND                  CREATED        STAT
+US        PORTS                      NAMES
+469a527e0016   quay.io/team-helium/miner:miner-amd64_2022.01.29.0_GA   "/opt/miner/bin/mine…"   21 hours ago   Up 2
+1 hours   127.0.0.1:4467->4467/tcp   miner
+```
+
+3. Check miner logs: `# less /var/miner_data/log/console.log`
+4. Check miner configs `# less /var/miner_config/sys.config`
+5. Purge miner
+
+```
+# docker kill miner
```

Comment:
This doesn't look right. That shoudl be below.

Reply:
Err ya confusing. Copied from shell prompt `root@snapshotter:/home/marvin#`. But `$` is more clear, will change.